vimeo-networking/src/main/java/com/vimeo/networking/logging/LoggingInterceptor.java

```diff
@@ -47,6 +49,9 @@
  */
 public class LoggingInterceptor implements Interceptor {
 
+    @NotNull
+    private final Gson gson = new GsonBuilder().setPrettyPrinting().create();
```

Comment:
Do we need `Gson` for this? If we use `Moshi` for this we could completely remove it as a dependency right?

Reply:
I can't find a way to do this with `Moshi`. There doesn't seem to be any printing options available, so I stuck with gson here. I'm thinking in the future we just remove this functionality from the library, since we mostly debug networking responses using other tools like Stetho.